nautobot_circuit_maintenance/urls.py

```diff
@@ -3,71 +3,25 @@
 from django.templatetags.static import static
 from django.urls import path
 from django.views.generic import RedirectView
-from nautobot.extras.views import ObjectChangeLogView
+from nautobot.core.views.routers import NautobotUIViewSetRouter
 
 from . import views
-from .models import CircuitImpact, CircuitMaintenance, Note, NotificationSource
 
 app_name = "nautobot_circuit_maintenance"
-# TODO: Uncomment and add to urlpatterns when converting to UIViewSets.
-# router = NautobotUIViewSetRouter()
+
+router = NautobotUIViewSetRouter()
+router.register("impact", views.CircuitImpactUIViewSet)
+router.register("maintenance", views.CircuitMaintenanceUIViewSet)
+router.register("note", views.NoteUIViewSet)
+router.register("rawnotification", views.RawNotificationUIViewSet)
+router.register("source", views.NotificationSourceUIViewSet)
```

Comment:
```suggestion
router.register("circuit-impact", views.CircuitImpactUIViewSet)
router.register("circuit-maintenance", views.CircuitMaintenanceUIViewSet)
router.register("note", views.NoteUIViewSet)
router.register("rawnotification", views.RawNotificationUIViewSet)
router.register("notification-source", views.NotificationSourceUIViewSet)
```

Reply:
```suggestion
router.register("circuit-impacts", views.CircuitImpactUIViewSet)
router.register("circuit-maintenances", views.CircuitMaintenanceUIViewSet)
router.register("notes", views.NoteUIViewSet)
router.register("raw-notifications", views.RawNotificationUIViewSet)
router.register("notification-sources", views.NotificationSourceUIViewSet)
```

Reply:
Making me eat my own words.. huh :) 

<img width="1105" height="584" alt="image" src="https://github.com/user-attachments/assets/61f65123-71ee-4bc9-b8da-d37b3572ab8f" />


I do think it makes sense to do a redirect, but I think fair enough to consider outside of scope.